spec/v1/digest_set.md

```diff
@@ -1,27 +1,29 @@
 # DigestSet field type specification
 
-Version: v1.0
+Version: v1.1
 
-Set of one or more cryptographic digests for a single software artifact or
-metadata object.
+Set of one or more immutable digests for a single software artifact or metadata
+object.
 
 ## Schema
 
 ```json
 {
-  "<ALGORITHM_1>": "<HEX/BASE64 VALUE>",
-  "<ALGORITHM_2>": "<HEX/BASE64 VALUE>",
-  ... 
+  "<ALGORITHM_1>": "<VALUE>",
+  "<ALGORITHM_2>": "<VALUE>",
+  ...
 }
 ```
 
 ## Fields
 
-A DigestSet is represented as a _JSON object_ mapping algorithm name to
-a string encoding of the digest using that algorithm. The named standard
-algorithms below use lowercase hex encoding. Usually there is just a
-single key/value pair, but multiple entries MAY be used for algorithm
-agility.
+A DigestSet is represented as a *JSON object* mapping algorithm name to a string
+encoding of the digest using that algorithm. The named standard algorithms below
+use lowercase hex encoding. Usually there is just a single key/value pair, but
+multiple entries MAY be used for algorithm agility.
+
+Users SHOULD use a *cryptographic* digest, but MAY use another identifier if the
+underlying implementation ensures immutability via other means.
```

Comment:
You might want to spell this out a bit more, since it's a bit nuanced. Maybe:

```suggestion
Each entry in a DigestSet MUST be an immutable reference to an artifact. It is
STRONGLY RECOMMENDED to use a commonly accepted, cryptographically secure digest
algorithm to achieve this immutability. See [Use cases for non-cryptographic,
immutable, digests](...) for further guidance.
```

Reply:
Done.